// @kustomer/apps-tokens/dist/constants/index.cjs
'use strict';

require('../chunk-AB4RIDZX.cjs');
var chunkQMO32RVY_cjs = require('../chunk-QMO32RVY.cjs');
var chunk6OB74BTN_cjs = require('../chunk-6OB74BTN.cjs');
var chunkSZCH5BGQ_cjs = require('../chunk-SZCH5BGQ.cjs');
var chunkIM2B63DK_cjs = require('../chunk-IM2B63DK.cjs');
var chunkT7NFNEYT_cjs = require('../chunk-T7NFNEYT.cjs');
var chunkMZRXPCSW_cjs = require('../chunk-MZRXPCSW.cjs');
var chunk3RVZVWGI_cjs = require('../chunk-3RVZVWGI.cjs');
var chunk4XWGRVTK_cjs = require('../chunk-4XWGRVTK.cjs');
var chunkZACPC22B_cjs = require('../chunk-ZACPC22B.cjs');
require('../chunk-QVDWLBP5.cjs');



Object.defineProperty(exports, 'sizes', {
	enumerable: true,
	get: function () { return chunkQMO32RVY_cjs.a; }
});
Object.defineProperty(exports, 'space', {
	enumerable: true,
	get: function () { return chunk6OB74BTN_cjs.a; }
});
Object.defineProperty(exports, 'transitions', {
	enumerable: true,
	get: function () { return chunkSZCH5BGQ_cjs.a; }
});
Object.defineProperty(exports, 'borderStyles', {
	enumerable: true,
	get: function () { return chunkIM2B63DK_cjs.a; }
});
Object.defineProperty(exports, 'fonts', {
	enumerable: true,
	get: function () { return chunkT7NFNEYT_cjs.e; }
});
Object.defineProperty(exports, 'globals', {
	enumerable: true,
	get: function () { return chunkMZRXPCSW_cjs.a; }
});
Object.defineProperty(exports, 'colors', {
	enumerable: true,
	get: function () { return chunk3RVZVWGI_cjs.a; }
});
Object.defineProperty(exports, 'radii', {
	enumerable: true,
	get: function () { return chunk4XWGRVTK_cjs.a; }
});
Object.defineProperty(exports, 'shadows', {
	enumerable: true,
	get: function () { return chunkZACPC22B_cjs.a; }
});
