// @kustomer/apps-common/dist/types/common/roles.cjs
'use strict';

var chunk7JYJLF25_cjs = require('../../chunk-7JYJLF25.cjs');
require('../../chunk-BJNRLTKL.cjs');



Object.defineProperty(exports, 'Attachment', {
	enumerable: true,
	get: function () { return chunk7JYJLF25_cjs.b; }
});
Object.defineProperty(exports, 'BasePermissions', {
	enumerable: true,
	get: function () { return chunk7JYJLF25_cjs.a; }
});
Object.defineProperty(exports, 'Company', {
	enumerable: true,
	get: function () { return chunk7JYJLF25_cjs.c; }
});
Object.defineProperty(exports, 'Conversation', {
	enumerable: true,
	get: function () { return chunk7JYJLF25_cjs.d; }
});
Object.defineProperty(exports, 'Customer', {
	enumerable: true,
	get: function () { return chunk7JYJLF25_cjs.e; }
});
Object.defineProperty(exports, 'Hook', {
	enumerable: true,
	get: function () { return chunk7JYJLF25_cjs.f; }
});
Object.defineProperty(exports, 'KObject', {
	enumerable: true,
	get: function () { return chunk7JYJLF25_cjs.h; }
});
Object.defineProperty(exports, 'KObjectCustom', {
	enumerable: true,
	get: function () { return chunk7JYJLF25_cjs.i; }
});
Object.defineProperty(exports, 'Klass', {
	enumerable: true,
	get: function () { return chunk7JYJLF25_cjs.g; }
});
Object.defineProperty(exports, 'Message', {
	enumerable: true,
	get: function () { return chunk7JYJLF25_cjs.j; }
});
Object.defineProperty(exports, 'Metadata', {
	enumerable: true,
	get: function () { return chunk7JYJLF25_cjs.k; }
});
Object.defineProperty(exports, 'Note', {
	enumerable: true,
	get: function () { return chunk7JYJLF25_cjs.l; }
});
Object.defineProperty(exports, 'Satisfaction', {
	enumerable: true,
	get: function () { return chunk7JYJLF25_cjs.m; }
});
Object.defineProperty(exports, 'Search', {
	enumerable: true,
	get: function () { return chunk7JYJLF25_cjs.n; }
});
